### PR TITLE
Fix/new mc 13– layout

### PR DESCRIPTION
**Pull request by MatejaC**

### CODE REVIEW

**Comment by e1401** [comment]
Screenshot 2024-01-06 at 11 56 32
We have a funny situation here, two branches with seemingly the same name but you have used em-dash instead of en-dash on the latter, the dash is longer, and therefore this warning 

Screenshot 2024-01-06 at 12 01 42

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -1,25 +1,21 @@
-import { Box, CssBaseline, Typography } from '@mui/material';
+import { Box, Typography } from '@mui/material';
 import { grey } from '@mui/material/colors';
 
 const Footer = () => {
```
Comment (e1401) [inline src/components/Footer.tsx:4]: consider 
```
const Footer = () => {
    return (
        // <Box
        //   sx={{
        //     display: 'flex',
        //     flexDirection: 'column',
        //     alignItems: 'center',
        //   }}
        // >
        <footer>
            <Box
                display='flex'
                flexDirection='column'
                justifyContent='center'
                alignItems='center'
                height='64px'
                border='1px solid red'
            >
                <Typography variant='caption' color={grey[400]}>
                    Made with ❤️ for colors and music
                </Typography>
            </Box>
        </footer>
    );
};

export default Footer;
```

We need to use semantic footer tag here

**Review by e1401 — CHANGES_REQUESTED** [review]
Three easy pieces

**Comment by e1401** [comment]
👋 bot hello

**Comment by e1401** [comment]
@MatejaC there are conlicts to be resolved in 
src/components/ResultCard.tsx
src/components/SearchBox.tsx
src/hooks/store.ts
yarn.lock

**Comment by MatejaC** [comment]
closing pull request, conflicts resolved, moving on

**Pull request closed without merging**